tools/prlint/module.ts

```diff
@@ -0,0 +1,37 @@
+import * as fs from 'fs-extra';
+import * as glob from 'glob';
+import * as path from 'path';
+
+const modules: string[] = [];
+
+export function findModulePath(fuzz: string): string {
+  if (modules.length === 0) {
+    const repoRoot = path.join(__dirname, '..', '..');
+    const lernaConfig = require(path.join(repoRoot, 'lerna.json'));
+    const searchPaths: string[] = lernaConfig.packages;
+    searchPaths.forEach(p => {
+      const globMatches = glob.sync(path.join(repoRoot, p));
+      const filtered = globMatches.filter(m => fs.existsSync(path.join(m, 'package.json')));
```

Comment:
can these two line be merged? using `glob.sync` to grep the full path using the `package.json` part?

Reply:
Yes! As a nice effect of this, this will reduce the number file access (disk IO) and make it more efficient. Thanks!